plugins/RoleMenuPlugin/Database/MediatR/GetRoleMenuRequest.cs

```diff
@@ -12,23 +13,23 @@ public static class GetRoleMenu
         ///     Retrieves a <see cref="RoleMenuModel" /> from the database in the form of a <see cref="RoleMenuModel" />
         /// </summary>
         /// <param name="MessageId">The Id of the message to grab.</param>
-        public sealed record Request(ulong MessageId) : IRequest<RoleMenuModel>;
+        public sealed record Request(ulong MessageId) : IRequest<Result<RoleMenuModel>>;
```

Comment:
Removing the `Request` suffix from the classes within the `MediatR` folder for the RoleMenuPlugin would make the naming consistent for the classes within the folder (not a big deal tho, just something small)